packages/databyss-ui/primitives/Modal/ModalView.js

```diff
@@ -10,6 +10,7 @@ const ModalView = ({
   secondaryChild,
   onDismiss,
   children,
+  disabled,
```

Comment:
This naming is confusing, because `disabled` on the modal would imply that you can't do/change anything in the modal. Maybe rename this prop to `canDismiss`? Keep in mind we are currently allowing a click outside of the modal to dismiss as well, so you would have to prevent that if `canDismiss` is false.